aarch64/src/l.S

```diff
@@ -0,0 +1,29 @@
+// Aarch64 entry (Raspberry Pi 3, 4 focussed)
+ 
+.section .boottext, "awx"
+.globl start
+
+start:
+	// All cores other than 0 should just hang
+    mrs     x1, mpidr_el1
```

Comment:
There's an odd mixture of hard-tabs and 4-space indent here. We should standardize. I vote for hard tabs for assembler, as homage to the past.

Reply:
fixed